.github/workflows/auto-labeler.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         id: extract_labels
         run: |
           # Supported labels definition
-          supported_labels=("ci" "doc" "core" "bug")
```

Comment:
Hi @Abdelhamid-khamis thanks for your contribution, we should not add all the labels to the list. We target only several labels. Can you please update the code to add only 3 new labels `enhancement`, `sample`, `usage` only?

Reply:
I committed the new change for this proposal in your branch. Going to merge now so you can have the contribution in this repo. Thanks